src/assets/i18n/fr.json5

```diff
@@ -6074,5 +6179,81 @@
 
   // "idle-modal.extend-session": "Extend session"
   "idle-modal.extend-session": "Prolonger la session"
```

Comment:
I think the issue is on this line... there should be a comma at the end of this line